bitwarden_license/src/Commercial.Core/SecretsManager/Commands/Secrets/CreateSecretCommand.cs

```diff
@@ -31,6 +31,11 @@ public async Task<Secret> CreateAsync(Secret secret, Guid userId)
             throw new NotFoundException();
         }
 
+        if (orgAdmin && secret.Projects != null && secret.Projects.Any() && !(await _projectRepository.ProjectsAreInOrganization(secret.Projects.Select(p => p.Id).ToList(), secret.OrganizationId)))
```

Comment:
I believe we want to check that all the projects are in the same organization as the secret, regardless if the user is org Admin or not.

```
if (secret.Projects != null && secret.Projects.Any() && !(await _projectRepository.ProjectsAreInOrganization(secret.Projects.Select(p => p.Id).ToList(), secret.OrganizationId)))
```

Reply:
Good find, I forgot to update this check in the Create. Updated.